### PR TITLE
Fix/testscenarios

### PR DESCRIPTION
**Pull request by sebolabs**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,6 @@
 *.plan
 *.tfplan
 .terraform
-
+**/terraform*
```
Comment (Zordrak) [inline .gitignore]: What's this for?

Comment (sebolabs) [reply .gitignore]: tfstate files exlusion

Comment (Zordrak) [reply .gitignore]: Can we replace with `*.tfstate*` instead?

Comment (sebolabs) [reply .gitignore]: of course, feel free to change it

**Pull request merged**